app/views/partials/paginated_selectable_team_members_table_form.scala.html

```diff
@@ -76,7 +77,7 @@
 @formWithCSRF(action = formAction) {
 
     @search_and_filter(form, withFilter =false, searchLabelKey = "group.members.list.filter.label")
-    <input type="hidden" name="pageSize" value="@paginationMetaData.get.pageSize"/>
+    <input type="hidden" name="pageSize" tm="@paginationMetaData.get.pageSize"/>
```

Comment:
How is this value used again? Not sure if you meant to change this

Reply:
oops